.env.example

```diff
@@ -1,11 +1 @@
-CENTRE_DETAILS_FILE_PATH=crawler/config/centre_details.json
-MONGO_DB=crawlerDevelopmentDB
-MONGO_HOST=127.0.0.1
-MONGO_PORT=27017
-SFTP_HOST=<sftp_host>
-SFTP_PASSWORD=<sftp_password>
-SFTP_PORT=22
-SFTP_USER=<sftp_user>
-SLACK_API_TOKEN=xoxb-123
-SLACK_CHANNEL_ID=Cxxx
-SETTINGS_MODULE=crawler.config.development
\ No newline at end of file
+SETTINGS_MODULE=crawler.config.development
```

Comment:
Could be worth adding LOCALHOST=host.docker.internal in here, as that is needed if you run this inside docker?

Reply:
But are you entering the virtual env (`pipenv shell`) in the containter?